custom/enikshay/case_utils.py

```diff
@@ -35,12 +35,11 @@ def get_parent_of_case(domain, case_id, parent_case_type):
 
```

Comment:
Farid and I noticed this bug here..
https://india.commcarehq.org/hq/admin/raw_doc/?id=39d45f04-3005-4ea1-8692-a5bf1e7ff685&db_name= 
references https://india.commcarehq.org/hq/admin/raw_doc/?id=190867c3-38db-4344-849f-c53849b252bd with incorrect case type. I am not sure if this is fixed now. Farid suggested this work around.